File: Server/profind-backend/src/main/java/com/profind/profind_backend/web/dto/ProjectDto.java
package com.profind.profind_backend.web.dto;

import java.time.Instant;
import java.util.List;

public class ProjectDto {
    public Long id;
    public Long professorId;
    public String title;
    public String shortDescription;
    public String description;
    public List<String> requirements;
    public List<String> tags;
    public String status;
    public Long proposalId;
    public Instant createdAt;
    public Instant updatedAt;
}

File: Interface/profind/src/class/Services/ProjectDto.ts
export interface Projectdto {
  id: number;
  title: string;
  shortDescription: string;
  description: string;
  requirements: string[];
  tags: string[];
  // status: "DRAFT" | "PENDING" | "APPROVED" | "REJECTED";
  status: string;
  proposalId: string;
  createdAt: string;
  updatedAt: string;
}
